base/resources/scripts/languagecheck.js

```diff
@@ -0,0 +1,40 @@
+/* Nodejs to set up a server on 8081 to modify the argv.json file
+   used by in-browser VSCode. Uses the accept-language header.
+   Also used by jupyter to determine language.
+ */
+var http = require("http");
+var fs = require("fs");
+
+http
+  .createServer(function (req, res) {
+    res.writeHead(200, { "Content-Type": "text/html" });
+    var lang = req.headers["accept-language"];
+    lang ? lang : (lang = "en");
+    lang = lang.substring(0, 2);
+
+    lang === "fr" || lang === "en" ? lang : (lang = "en");
+
+    // READ THE FILE AND COMPARE TO LANG IF IT CHANGES DO THE FOLLOWING, SYNC to not write to it as well
+    var file = fs.readFileSync(
```

Comment:
* Don't use `var` unless you mean to. Prefer `const` if the variable will not be reassigned, or `let` if it will.
* Careful on variable names; `file` suggests that the variable holds a file ref rather than its string contents.